### PR TITLE
add delete button to task

### PR DESCRIPTION
**Pull request by ChristinaPastushchak**
## JIRA

https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777&view=detail&selectedIssue=LEAR-27

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [x] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/event/EventService.js [hunk 1]
```diff
@@ -63,3 +63,4 @@ export const PostEventService = (data) => {
     const url = teamPath + data.team + '/event/';
     axios.post(url, data);
 };
+export const deleteTeamService = (eventId,taskId) =>axios.delete(eventPath + eventId + '/task/'+taskId + '/');
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/event/EventService.js]: add space after => and around plus sign in '/task/'+taskId

Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/event/EventService.js]: ...and space before taskId in (eventId,taskId)

File: eventually/static/src/containers/event/EventTaskItem.js [hunk 2]
```diff
@@ -6,18 +6,25 @@ import EditEventTaskDialog from './EditEventTaskDialog';
...
 import { withRouter } from 'react-router-dom';
 import Task from '../taskItem/Task';
+import {getEvents,deleteTeamService} from 'src/containers/event/EventService';
 
 
 export default class EventTaskItem extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
+            id:this.props.id,
```
Comment (VasilHamulyak) [inline eventually/static/src/containers/event/EventTaskItem.js]: Add space after id:

File: eventually/static/src/containers/event/EventTaskItem.js [hunk 3]
```diff
@@ -6,18 +6,25 @@ import EditEventTaskDialog from './EditEventTaskDialog';
 import { Link } from 'react-router';
 import { withRouter } from 'react-router-dom';
 import Task from '../taskItem/Task';
+import {getEvents,deleteTeamService} from 'src/containers/event/EventService';
```
Comment (VasilHamulyak) [inline eventually/static/src/containers/event/EventTaskItem.js]: Add space after and before curly braces, also after commas

File: eventually/static/src/containers/event/EventTaskItem.js [hunk 4]
```diff
@@ -6,18 +6,25 @@ import EditEventTaskDialog from './EditEventTaskDialog';
...
         this.state = {
+            id:this.props.id,
             eventId: this.props.eventId,
             title: this.props.title,
             description: this.props.description.slice(0,300)+'...',
             openDialog: false
         };
     }
+    handleDelete= () =>{
+        deleteTeamService(this.state.eventId,this.state.id);
```
Comment (VasilHamulyak) [inline eventually/static/src/containers/event/EventTaskItem.js]: Add space after commas

File: eventually/static/src/containers/event/EventService.js [hunk 5]
```diff
@@ -63,3 +63,4 @@ export const PostEventService = (data) => {
     const url = teamPath + data.team + '/event/';
     axios.post(url, data);
 };
+export const deleteTeamService = (eventId, taskId) => axios.delete(eventPath + eventId + '/task/'+taskId + '/');
```
Comment (lhalam) [inline eventually/static/src/containers/event/EventService.js]: View the whole file!
Correct according to the convention!

File: eventually/static/src/containers/event/EventTaskItem.js [hunk 6]
```diff
@@ -6,18 +6,25 @@ import EditEventTaskDialog from './EditEventTaskDialog';
...
+    }
+
```
Comment (lhalam) [inline eventually/static/src/containers/event/EventTaskItem.js]: excess the lines

**Review by lhalam — DISMISSED** [review]

**Review by VasilHamulyak — DISMISSED** [review]

**Review by tarassito — APPROVED** [review]

**Pull request merged**